testing/src/test/java/org/motechproject/nms/testing/it/api/MobileAcademyControllerBundleIT.java

```diff
@@ -147,7 +141,7 @@ public void testGetCourseValid() throws IOException, InterruptedException {
         HttpGet request = RequestBuilder.createGetRequest(endpoint);
 
         HttpResponse httpResponse = SimpleHttpClient.httpRequestAndResponse(request, RequestBuilder.ADMIN_USERNAME, RequestBuilder.ADMIN_PASSWORD);
-        assertTrue(httpResponse.getStatusLine().getStatusCode() == 200);
+        assertEquals(200, httpResponse.getStatusLine().getStatusCode());
```

Comment:
But the problem here is you don't know what the returned value was, you just know it wasn't 200.
This tells you something like "expected 200 but received 404"
~is better no?~